core/src/migrations/000085-addRecordsPropertyInvalid.ts

```diff
@@ -0,0 +1,17 @@
+import Sequelize from "sequelize";
+
+export default {
+  up: async (
+    queryInterface: Sequelize.QueryInterface,
+    DataTypes: typeof Sequelize
+  ) => {
+    await queryInterface.addColumn("records", "invalid", {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+    });
```

Comment:
If you add a new column in a migration that is non-null, you'll need to immediately then set the existing rows to /something/.